src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineTypeSection.qml

```diff
@@ -91,21 +90,44 @@ Column {
             navigationRowStart: startHookButtonGroup.navigationRowEnd + 1
         }
 
+    Item {
+        height: childrenRect.height
+        width: parent.width
+
+        SpinBoxPropertyView {
+            id: startHookHeightSection
+            anchors.left: parent.left
+            anchors.right: parent.horizontalCenter
+            anchors.rightMargin: 2
+
+            titleText: qsTrc("inspector", "Start Hook height")
```

Comment:
The preferred spelling would be "Start <i>h</i>ook height", i.e. with lowercase _h_ in _hook_.

(Same for "End <I>h</I> hook height" a few lines below)

Reply:
Thanks for the catch! I copied from the camel-case variable names and didn't look closely